package.json

```diff
@@ -47,12 +47,10 @@
     "@backstage/core-plugin-api": "^1.0.6",
     "@backstage/plugin-catalog-react": "^1.1.4",
     "@backstage/theme": "^0.2.16",
-    "@cortexapps/backstage-plugin-extensions": "^0.0.11",
+    "@cortexapps/backstage-plugin-extensions": "0.0.12",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "4.0.0-alpha.45",
-    "@mui/icons-material": "^5.0.4",
-    "@mui/material": "^5.0.4",
```

Comment:
This kind of feels like a reversion 🙃. [MUI is material-ui v5](https://mui.com/blog/material-ui-is-now-mui/) (at some point we really should fully upgrade).

Reply:
yupp agreed, we just have a dependency we're not using at this point 🤷🏽 material-ui -> mui should be done as its own change

Reply:
also backstage itself is still on `material-ui@4.x`

Reply:
> also backstage itself is still on `material-ui@4.x`

Oof good find! 😆